apps/fluent-tester/src/TestComponents/CheckboxExperimental/CheckboxTest.tsx

```diff
@@ -21,9 +23,12 @@ const BasicCheckbox: React.FunctionComponent = () => {
       <Checkbox label="Checked checkbox (uncontrolled)" onChange={onChangeUncontrolled} defaultChecked accessibilityLabel="Hello there" />
       <Checkbox label="Disabled checkbox" disabled />
       <Checkbox label="Disabled checked checkbox" defaultChecked disabled />
-      <Checkbox label="Checkbox will display a tooltip" tooltip="This is a tooltip" />
-      <Checkbox label="A circular checkbox" shape="circular" />
-      <Checkbox label="A checkbox with label placed before" labelPosition="before" />
+
+      {/* No Circular Checkbox , Tooltip , Before Label Support on Android */}
+
+      {isAndroid ? <></> : <Checkbox label="Checkbox will display a tooltip" tooltip="This is a tooltip" />}
+      {isAndroid ? <></> : <Checkbox label="A circular checkbox" shape="circular" />}
+      {isAndroid ? <></> : <Checkbox label="A checkbox with label placed before" labelPosition="before" />}
```

Comment:
Could we perhaps put the stuff not supported in Android into it's own section and render some text as well? Or have a better way to tell the user not to use this variant (like a warning)?

Reply:
I have an idea of what to do, I'll try to spin up a proposal in the next week. I guess for now, this is not blocking

Reply:
@ankraj12 , Yes , moving ahead with the current code and logging item for this to tackle , when @Saadnajmi proposal is in.